Index: Assets/TextMesh Pro.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/TextMesh Pro.meta b/Assets/TextMesh Pro.meta
deleted file mode 100644
--- a/Assets/TextMesh Pro.meta	(revision c9982613d51cdaa2f702b6242494d8085e59edb7)
+++ /dev/null	(revision c9982613d51cdaa2f702b6242494d8085e59edb7)
@@ -1,8 +0,0 @@
-fileFormatVersion: 2
-guid: f54d1bd14bd3ca042bd867b519fee8cc
-folderAsset: yes
-DefaultImporter:
-  externalObjects: {}
-  userData: 
-  assetBundleName: 
-  assetBundleVariant: 
